### PR TITLE
Update --help output for workflow commands

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Updated some of the --help messages based on https://docs.temporal.io/docs/tctl/workflow

## Why?

fixing --help output spelling, such as `--reset-reapply-type value    ... Support one of these: ,Signal,NoneDefault to: All`

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: cli/workflow.go [hunk 1]
```diff
@@ -123,15 +123,15 @@ func newWorkflowCommands() []*cli.Command {
 		},
 		{
 			Name:  "scan",
-			Usage: "Scan workflow executions (requires Elasticsearch to be enabled)",
+			Usage: "List workflow executions. Faster and unsorted (requires Elasticsearch to be enabled)",
```
Comment (DJSanti) [inline cli/workflow.go:126]: What is it faster than?

File: cli/workflow.go [hunk 2]
```diff
@@ -192,58 +192,52 @@ func newWorkflowCommands() []*cli.Command {
...
-				&cli.StringFlag{
-					Name:    FlagInputFile,
-					Aliases: FlagInputFileAlias,
-					Usage:   "Input file to use for resetting, one workflow per line of WorkflowId and RunId. RunId is optional, default to current runId if not specified. ",
-				},
 				&cli.StringFlag{
 					Name:    FlagListQuery,
 					Aliases: FlagListQueryAlias,
-					Usage:   "Visibility query to get workflows to reset",
+					Usage:   "Visibility query of Search Attributes describing the Workflow Executions to reset",
```
Comment (bergundy) [inline cli/workflow.go:201]: Maybe link to the page that explains the query syntax?

File: cli/workflow.go [hunk 3]
```diff
@@ -123,15 +123,15 @@ func newWorkflowCommands() []*cli.Command {
...
+			Usage: "List workflow executions. Faster and unsorted (requires Elasticsearch to be enabled)",
 			Flags: append(flagsForScan, flags.FlagsForPaginationAndRendering...),
 			Action: func(c *cli.Context) error {
 				return ScanAllWorkflow(c)
 			},
 		},
 		{
 			Name:  "count",
-			Usage: "Count number of workflow executions (need to enable Temporal server on ElasticSearch)",
+			Usage: "Count workflow executions (requires ElasticSearch to be enabled)",
```
Comment (DJSanti) [inline cli/workflow.go:134]: If there's more than one of these notes about ElasticSearch, I would suggest moving it to its own ::: note block above the commands it pertains to.

File: cli/workflow.go [hunk 4]
```diff
@@ -147,12 +147,12 @@ func newWorkflowCommands() []*cli.Command {
 		},
 		{
 			Name:  "terminate",
-			Usage: "Terminate a new workflow execution",
+			Usage: "Terminate a workflow execution",
```
Comment (DJSanti) [inline cli/workflow.go:150]:
```suggestion
			Usage: "Ends a workflow execution",
```

Comment (DJSanti) [reply cli/workflow.go:150]: or end? Also, is it a given WF execution, or the most recently created?

File: cli/workflow.go [hunk 5]
```diff
@@ -192,58 +192,52 @@ func newWorkflowCommands() []*cli.Command {
...
-				},
 				&cli.StringFlag{
 					Name:    FlagListQuery,
 					Aliases: FlagListQueryAlias,
-					Usage:   "Visibility query to get workflows to reset",
+					Usage:   "Visibility query of Search Attributes describing the Workflow Executions to reset",
+				}, &cli.StringFlag{
+					Name:    FlagInputFile,
+					Aliases: FlagInputFileAlias,
+					Usage:   "Input file that specifies Workflow Executions to reset. Each line contains one Workflow Id as the base Run and, optionally, a Run Id",
```
Comment (bergundy) [inline cli/workflow.go:205]: I'd explain what the format is (`workflowId,runId`)?
Can this read from stdin if file is `-`? That could be useful so you can list workflows | reset.

File: cli/workflow.go [hunk 6]
```diff
@@ -192,58 +192,52 @@ func newWorkflowCommands() []*cli.Command {
...
 					Name:  FlagExcludeFile,
 					Value: "",
-					Usage: "Another input file to use for excluding from resetting, only workflowId is needed",
+					Usage: "Input file that specifies Workflow Executions to exclude from resetting",
 				},
 				&cli.StringFlag{
 					Name:  FlagInputSeparator,
 					Value: "\t",
-					Usage: "Separator for input file(default to tab)",
+					Usage: "Separator for the input file. The default is a tab (\t)",
```
Comment (bergundy) [inline cli/workflow.go:215]: Oh I see this here..

File: cli/workflow.go [hunk 7]
```diff
@@ -147,12 +147,12 @@ func newWorkflowCommands() []*cli.Command {
...
 		{
 			Name:  "terminate",
-			Usage: "Terminate a new workflow execution",
+			Usage: "Terminate a workflow execution",
 			Flags: append(flagsForExecution, []cli.Flag{
 				&cli.StringFlag{
 					Name:    FlagReason,
 					Aliases: FlagReasonAlias,
-					Usage:   "The reason you want to terminate the workflow",
+					Usage:   "Reason for terminating the workflow execution",
```
Comment (DJSanti) [inline cli/workflow.go:155]:
```suggestion
					Usage:   "Reason for terminating the Workflow Execution",
```

File: cli/workflow.go (hunk 4 above)
Comment (DJSanti) [reply cli/workflow.go:150]: Capitalize the term

File: cli/workflow.go (hunk 3 above)
Comment (DJSanti) [inline cli/workflow.go:134]:
```suggestion
			Usage: "Count Workflow Executions (requires ElasticSearch to be enabled)",
```

File: cli/workflow.go (hunk 1 above)
Comment (DJSanti) [inline cli/workflow.go:126]:
```suggestion
			Usage: "List Workflow Executions. Faster and unsorted (requires Elasticsearch to be enabled)",
```

File: cli/workflow.go [hunk 8]
```diff
@@ -169,18 +169,18 @@ func newWorkflowCommands() []*cli.Command {
 				},
 				&cli.StringFlag{
 					Name:     FlagReason,
-					Usage:    "reason to do the reset",
+					Usage:    "Reason for resetting the workflow execution",
```
Comment (DJSanti) [inline cli/workflow.go:172]:
```suggestion
					Usage:    "Reason for resetting the Workflow Execution",
```

File: cli/workflow.go [hunk 9]
```diff
@@ -169,18 +169,18 @@ func newWorkflowCommands() []*cli.Command {
...
 					Name: FlagResetType,
-					Usage: "Where to reset. Support one of these: " +
+					Usage: "Event type to which you want to reset: " +
 						strings.Join(mapKeysToArray(resetTypesMap), ","),
 				},
 				&cli.StringFlag{
 					Name: FlagResetReapplyType,
-					Usage: "Whether to reapply events after the reset point. Support one of these: " +
-						strings.Join(mapKeysToArray(resetReapplyTypesMap), ",") + "Default to: All",
+					Usage: "Types of events to reapply after the reset point: " +
```
Comment (DJSanti) [inline cli/workflow.go:182]:
```suggestion
					Usage: "Event types to reapply after the reset point: " +
```

File: cli/workflow.go [hunk 10]
```diff
@@ -192,58 +192,52 @@ func newWorkflowCommands() []*cli.Command {
...
 				&cli.IntFlag{
 					Name:  FlagParallelism,
 					Value: 1,
-					Usage: "Number of goroutines to run in parallel. Each goroutine would process one line for every second",
+					Usage: "Number of goroutines to run in parallel. Each goroutine processes one line for every second",
 				},
 				&cli.BoolFlag{
 					Name:  FlagSkipCurrentOpen,
-					Usage: "Skip the workflow if the current run is open for the same workflowId as base",
+					Usage: "Skip a workflow execution if the current Run is open for the same Workflow Id as the base Run",
```
Comment (DJSanti) [inline cli/workflow.go:229]:
```suggestion
					Usage: "Skip a Workflow Execution if the current Run is open for the same Workflow Id as the base Run",
```

File: cli/workflow.go [hunk 11]
```diff
@@ -192,58 +192,52 @@ func newWorkflowCommands() []*cli.Command {
...
+					Usage: "Skip a Workflow Execution if the base Run is not the current Run",
 				},
 				&cli.BoolFlag{
 					Name:  FlagNonDeterministic,
-					Usage: "Only apply onto workflows whose last event is workflowTaskFailed with non deterministic error",
-				},
-				&cli.BoolFlag{
-					Name:  FlagDryRun,
-					Usage: "Not do real action of reset(just logging in STDOUT)",
+					Usage: "Reset workflow execution only if its last event is WorkflowTaskFailed with a nondeterministic error",
```
Comment (DJSanti) [inline cli/workflow.go:240]:
```suggestion
					Usage: "Reset Workflow Execution only if its last Event is WorkflowTaskFailed with a nondeterministic error",
```

**Review by DJSanti — APPROVED** [review]
There's some capitalization issues and a few other terms that could be made clearer. Fix that, and you're good to go.

**Review by bergundy — APPROVED** [review]
Had a couple of suggestions

File: cli/workflow.go (hunk 2 above)
Comment (feedmeapples) [reply cli/workflow.go]: done

File: cli/workflow.go (hunk 1 above)
Comment (feedmeapples) [reply cli/workflow.go]: than `workflow list`, though not mentioned here. --help outputs this:
```
tctl w -h
...

COMMANDS:
   run          Start a new workflow execution and show progress
   describe, d  Show information of workflow execution
   list, l      List workflow executions based on query
   show         Show workflow history
   query        Query workflow execution
   stack        Query workflow execution with __stack_trace as query type
   signal, s    Signal a workflow execution
   scan         List Workflow Executions. Faster and unsorted (requires Elasticsearch to be enabled)
   count        Count Workflow Executions (requires ElasticSearch to be enabled)
```

**Pull request merged**